doc/usage/extensions/napoleon.rst

```diff
@@ -542,6 +565,8 @@ sure that "sphinx.ext.napoleon" is enabled in ``conf.py``::
    .. versionadded:: 3.2
 
 .. confval:: napoleon_attr_annotations
+   :type: `bool`
```

Comment:
Using backticks can make text appear more stylized. Should we apply them in other contexts as well?

Reply:
Sad, github action tell me `"default role used (hint: for inline literals, use double backticks) (default-role)"`. But when I use double backticks, the inline literals looks not good as I want.

Though I see the early documentation used double backticks also.

Reply:
Already update to double backticks.

Reply:
Inspired by the below code, I add the `code-py` interpreted text role.
https://github.com/sphinx-doc/sphinx/blob/230ccf2a44d750f5d6c5542f970f48ffe0cbce1e/doc/usage/configuration.rst?plain=1#L2917-L2919